owners/info_server.js

```diff
@@ -176,32 +173,10 @@ class InfoServer extends Server {
 }
 
 if (require.main === module) {
-  require('dotenv').config();
-
-  const Octokit = require('@octokit/rest');
```

Comment:
Same question: Can this be removed from `package.json`?

Reply:
Same answer: no, it's used in `bootstrap`, that's why it's removed from these duplicate code blocks.